accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/api/InstallationLog.java

```diff
@@ -8,35 +8,26 @@
  */
 package biz.netcentric.cq.tools.actool.api;
 
-import org.osgi.annotation.versioning.ConsumerType;
+import java.util.Set;
 
-import biz.netcentric.cq.tools.actool.installationhistory.AcInstallationHistoryPojo;
+import org.osgi.annotation.versioning.ProviderType;
 
 /**
  * Access to log messages being emitted
  *
  */
-@ConsumerType
-public interface InstallationLog extends AcInstallationHistoryPojo {
+@ProviderType
```

Comment:
not sure about this change. Is it reasonable for consumers to implement this?

Reply:
no, the change makes sense to me (was wrong in the first place)